lib/json_to_graphql/parser.rb

```diff
@@ -22,9 +22,19 @@ def build_nodes(hash, parent_node)
     end
 
     def print
-      root.children.each_with_object("{\n") do |child, query|
-        query.concat("#{child.print("", 1)}")
-      end.concat("}")
+      if input_hash.key?('query')
+        start_string = ""
+        start_space = 0
+      else
+        start_string = "{\n"
+        start_space = 1
+      end
+
+      query = root.children.each_with_object(start_string) do |child, query|
+        query.concat("#{child.print("", start_space)}")
+      end
+      query.concat("}") unless input_hash.key?('query')
```

Comment:
I think this should include `query`, `mutation`, or `subscription`. All of these could be operation types.